scripts/example.ts

```diff
@@ -0,0 +1,144 @@
+import path from 'path';
+import { remove, pathExists } from 'fs-extra';
+import prompts from 'prompts';
+
+import { getOptionsOrPrompt } from './utils/options';
+import type { CLIStep } from './utils/cli-step';
+import { executeCLIStep } from './utils/cli-step';
+
+const frameworks = ['react'];
+const addons = ['a11y', 'storysource'];
+const examplesDir = path.resolve(__dirname, '../examples');
+
+async function getOptions() {
+  return getOptionsOrPrompt('yarn example', {
+    framework: {
+      description: 'Which framework would you like to use?',
+      values: frameworks,
+      required: true,
+    },
+    addon: {
+      description: 'Which extra addons (beyond the CLI defaults) would you like installed?',
+      values: addons,
+      multiple: true,
+    },
+    includeStories: {
+      description:
+        "Include Storybook's own stories (only applies if a react-based framework is used)?",
+    },
+    create: {
+      description: 'Create the example from scratch (rather than degitting it)?',
+    },
+    forceDelete: {
+      description: 'Always delete an existing example, even if it has the same configuration?',
+    },
+    forceReuse: {
+      description: 'Always reusing an existing example, even if it has a different configuration?',
+    },
+    verdaccio: {
+      description: 'Use verdaccio rather than yarn linking stories?',
+    },
+    start: {
+      description: 'Start the example app?',
+      inverse: true,
+    },
+    build: {
+      description: 'Build the example app?',
+    },
+    watch: {
+      description: 'Start building used packages in watch mode as well as the example app?',
+    },
+    dryRun: {
+      description: "Don't execute commands, just list them?",
```

Comment:
This could probably not be a question, but just a flag? Most people who will use this command will not want to run it in dry run mode anyway, it's mostly for maintenance purposes.

Or at least mention (dry run) to make this more explicit.

```suggestion
      description: "Don't execute commands, just list them (dry run)?",
```